src/chi-vue/src/views/DataTable/ClientSide/DataTableClientView.vue

```diff
@@ -89,6 +89,13 @@
           </template>
         </ChiDataTableToolbar>
       </template>
+      <template #bulkActions>
+        <ChiDataTableBulkActions
+          :bulkActionConfig="bulkActionConfig"
+          @chiShowSelectedOnly="e => chiShowSelectedOnly(e)"
+        >
```

Comment:
The hardcoded actions buttons must be provided from here using templates and slots.